config.default.yaml

```diff
@@ -98,25 +106,27 @@ electricity:
     epsilon_vres: 0.02 # share of total renewable supply
     contingency: 0 # fixed capacity in MW
 
-  extendable_carriers:
-    Generator: []
-    StorageUnit: []
-    Store: [battery, H2]
-    Link: []
-
-  conventional_carriers: [coal, OCGT, CCGT, oil]
-
   max_hours:
     battery: 6
     H2: 168
 
-  powerplants_filter:
+  extendable_carriers:
+    Generator: [solar, onwind, offwind-ac, offwind-dc, OCGT]
+    StorageUnit: [] # battery, H2
+    Store: [battery, H2]
+    Link: [AC, DC]
```

Comment:
We have to understand the behavior of this PR combined with the one with HVDC lines implemented as dc lines

Reply:
Yes, once @ekatef PR is merged we have to revise quite a bit here
I would take care of it & potentially discuss changes with @ekatef

Btw, I think here the word link refers rather to what "network" is included